pyglui/ui_elements.pxi

```diff
@@ -5,7 +5,7 @@ cdef class UI_element:
     '''
     The base class for all UI elements.
     '''
-    cdef readonly bytes label
+    cdef readonly object label
```

Comment:
Why do you use `object` in this case and not `str`? Might be more consistent with the rest of the file.